src/ssl.c

```diff
@@ -2829,6 +2829,60 @@ const WOLFSSL_EVP_CIPHER *wolfSSL_EVP_get_cipherbyname(const char *name)
     return NULL;
 }
 
+const WOLFSSL_EVP_CIPHER *wolfSSL_EVP_get_cipherbynid(int id)
```

Comment:
Is this enough? 

/*
 * return EVP_CIPHER structure by NID
 *
 * paremeter id  cipher NID
 *
 * retrun WOLFSSL_EVP_CIPHER
*/